code/modules/mob/living/simple_animal/abnormality/waw/red_riding_mercenary.dm

```diff
@@ -0,0 +1,688 @@
+		"What's the matter, scared?",
+		"Get your eye checked."
+	)
+
+	var/list/weapon_throw_lines = list( // Using the weapon throw attack.
+		"No hesitation!",
+		"You're dead!",
+		"Eat this!"
+	)
+
+/datum/action/innate/abnormality_attack/find_target // AI-controlled Red technically doesn't use this one.
+	name = "Locate target"
+	icon_icon = 'ModularTegustation/Teguicons/teguicons.dmi'
+	button_icon_state = "red_target"
+	chosen_message = "<span class='danger'>You try to suss out where your target is...</span>"
+	chosen_attack_num = 1
+
+/datum/action/innate/abnormality_attack/find_target/Activate()
+	addtimer(CALLBACK(A, .mob/living/simple_animal/hostile/abnormality/red_hood/proc/PlayerTargetFind), 1)
+	to_chat(A, chosen_message)
+
+/mob/living/simple_animal/hostile/abnormality/red_hood/proc/PlayerTargetFind()
+	HunterTracking(priority_target) // basically a redirect, ability code is weird
+
+/datum/action/innate/abnormality_attack/catch_breath // AI-controlled Red technically doesn't use this one EITHER.
+	name = "Evade"
+	icon_icon = 'ModularTegustation/Teguicons/teguicons.dmi'
+	button_icon_state = "ruina_evade"
+	chosen_message = "<span class='danger'>You prepare to avoid an incoming attack.</span>"
+	chosen_attack_num = 2
+
+/datum/action/innate/abnormality_attack/catch_breath/Activate()
+	addtimer(CALLBACK(A, .mob/living/simple_animal/hostile/abnormality/red_hood/proc/AttemptEvade), 1)
+	to_chat(A, chosen_message)
+
+/mob/living/simple_animal/hostile/abnormality/red_hood/proc/AttemptEvade()
+	if((world.time < evade_timer) || evading_attack)
+		if(client)
+			to_chat(src, "<span class='danger'> You can't do that now!</span>")
+		return FALSE
+	evading_attack = TRUE
+	addtimer(CALLBACK(src, .proc/EndEvade), 20)
+	return
+
+/mob/living/simple_animal/hostile/abnormality/red_hood/apply_damage(damage = 0,damagetype = RED_DAMAGE, def_zone = null, blocked = FALSE, forced = FALSE, spread_damage = FALSE, wound_bonus = 0, bare_wound_bonus = 0, sharpness = SHARP_NONE, white_healable = FALSE)
+	if(evading_attack)
+		evading_attack = FALSE
+		EndEvade()
+		return FALSE
+	..()
+	if(health > (maxHealth * 0.8) || !priority_target)
+		return
+	if(red_rage < 1 && target_priority > 2)
+		RageUpdate(1)
+
+/mob/living/simple_animal/hostile/abnormality/red_hood/proc/WatchIt() // Evade Blue's indiscriminate attacks, just to fuck him even harder.
+	if(red_rage < 2)
+		if(target_priority != 1)
+			say(pick(blue_evade_lines))
+		else
+			say(pick(blue_evade_taunt_lines))
+	else
+		if(!client)
+			manual_emote("acrobatically spins out of the way.")
+	SpinAnimation(7, 1)
+	adjustBruteLoss(-50)
+	return
+
+/mob/living/simple_animal/hostile/abnormality/red_hood/proc/RageUpdate(rage_change) // Always call this after changing red_rage manually, or call it to change red_rage
+	if(rage_change)
+		red_rage = rage_change
+	if(red_rage > 0)
+		damage_coeff = list(RED_DAMAGE = 0.3, WHITE_DAMAGE = 0.6, BLACK_DAMAGE = 0.3, PALE_DAMAGE = 1.5) // She takes... very little damage.
```

Comment:
This still won't work, it needs to use ChangeResistances() to edit the base value, as damage_coeff, like armor, becomes a datum on Initialize()